mpt/impl/imap-mailbox/core/src/main/resources/org/apache/james/imap/scripts/Annotation.test

```diff
@@ -61,3 +70,76 @@ S: a13 BAD SETMETADATA failed. Illegal arguments.
 
 C: a14 SETMETADATA INBOX (/private/comment My new comment)
 S: a14 BAD SETMETADATA failed. Illegal arguments.
+
+########################For get annotation############
+
+C: s1 SETMETADATA INBOX (/private/comment "My own comment")
+S: s1 OK SETMETADATA completed.
+
+C: s2 SETMETADATA INBOX (/shared/comment "The shared comment")
+S: s2 OK SETMETADATA completed.
+
+C: g3 GETMETADATA "INBOX" (/shared/comment /private/comment)
+S: \* METADATA "INBOX" \(\/private\/comment "My own comment" \/shared\/comment "The shared comment"\)
+S: g3 OK GETMETADATA completed.
+
+C: g4 GETMETADATA "INBOX"
+S: \* METADATA "INBOX" \(\/private\/comment "My own comment" \/shared\/comment "The shared comment"\)
+S: g4 OK GETMETADATA completed.
+
+C: g5 GETMETADATA "INBOX" /shared/comment /private/comment)
+S: g5 BAD GETMETADATA failed. Illegal arguments.
+
+C: g6 GETMETADATA "INBOX" (/shared/comment /private/comment
+S: g6 BAD GETMETADATA failed. Illegal arguments.
+
+C: g7 GETMETADATA "INBOX" private/comment
+S: \* METADATA "INBOX"
+S: g7 OK GETMETADATA completed.
+
+C: g8 GETMETADATA "INBOX" (MAXSIZE 15)
+S: \* METADATA "INBOX" \(\/private\/comment "My own comment"\)
+S: g8 OK \[METADATA LONGENTRIES 18\] GETMETADATA completed.
```

Comment:
What happen if you have equality between size of a metadata and the MAXSIZE value ?

We need a test about this. Here and in the unit tests.